pyproject.toml

```
@@ -40,8 +40,8 @@ dependencies = [
     "einops>=0.3",
     # fiona 1.8.21+ required for Python 3.10 wheels
     "fiona>=1.8.21",
-    # kornia 0.7.3+ required for instance segmentation support in AugmentationSequential
-    "kornia>=0.7.3",
+    # kornia 0.7.4+ required for using native AugmentationSequential, allowing us to replace our custom implementation
```

```suggestion
    # kornia 0.7.4+ required for AugmentationSequential support for unknown keys
```
No need to mention something that will no longer exist